5calls/app/src/main/res/layout/activity_about.xml

```diff
@@ -1,177 +1,76 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/background_color"
-    >
+    android:background="@color/background_color">
```

Comment:
Normally I like to wrap the closing > or /> in the XML to the next line, to reduce diffs when adding new lines to the XML.

You don't have to do this but I'm just mentioning it as an optional comment!